### DISCUSSION TITLE
How do you retrieve profile avatars if you accidentally deleted the cache directory?

### DISCUSSION THREAD

**Discussion by nickali** [mastodon/mastodon#22820] [Q&A]
When moving around a mastodon install to a different server, the cache directory was deleted. 

I can see avatars for people I started following after the move, but not for the ones before the move.

Is there some way to trigger downloading the avatars again?

**Comment by finneh4249** [comment]
You can use tootctl to refresh avatars of remote users but as far as I know, any local content that was deleted is gone.
`tootctl accounts refresh`
